assets/src/edit-story/elements/media/edit.js

```diff
@@ -224,11 +225,18 @@ function MediaEdit({ element, box }) {
           srcSet={calculateSrcSet(resource)}
         />
       )}
-      {isVideo && (
+      {isVideo && resource.src && (
         <FadedVideo {...fadedMediaProps}>
           <source src={resource.src} type={resource.mimeType} />
         </FadedVideo>
       )}
+      {isVideo && !resource.src && (
```

Comment:
```suggestion
      {isVideo && !resource.src && resource.poster && (
```

You also need to check the poster is set here. Otherwise, you get an invalid image tag. If poster also not set, should we put the placeholder grey in?